client.js

```diff
@@ -39,4 +43,25 @@ const employees = [
 // This is not a race. Everyone on your team should understand what is happening.
 // Ask questions when you don't.
 
-console.log( employees );
+
+
+
+// Loop over the employees array
+/*
+for(let i; i < ??; i++) {
+  // Call the function below, passing a single employee object as the parameter
+  // You will get back (see the return in the function) a new object. Log that new object to the console.
+}
+*/
+
+
+
+// this function will calculate 1 employee's bonus!
+function calculateIndividualEmployeeBonus( employee ) {
```

Comment:
We'll need to update the readme, to match this. Students definitely get confused by the instructions. eg:

```md
## Processing Employee Bonuses

Loop over the `employees` array and do the following for each employee object:

* Call the `calculateIndividualEmployeeBonus` function, and pass the employee object as an argument.
* Assign the return value of `calculateIndividualEmployeeBonus()` to a variable
* Log the value of that variable
```

and

```md
## Function Logic

The `calculateIndividualEmployeeBonus` function receives a single `employee` object as an argument. 
You will need to calculate that employee's bonus, based on what you know about them. 
The function should then return a new **object** with the following properties:
```

Reply:
I'm going to rework the instructions and the code comments with the above in mind. The instructions are pretty explicit, I feel, yet students either don't read them or try to move too quickly. I want to avoid having the code comments just restate the instructions